src/modules/powerrename/lib/trace.cpp

```diff
@@ -0,0 +1,73 @@
+#include "stdafx.h"
+#include "trace.h"
+
+TRACELOGGING_DEFINE_PROVIDER(
+      g_hProvider,
+      "Microsoft.PowerToys",
+      // {38e8889b-9731-53f5-e901-e8a7c1753074}
+      (0x38e8889b, 0x9731, 0x53f5, 0xe9, 0x01, 0xe8, 0xa7, 0xc1, 0x75, 0x30, 0x74),
+      TraceLoggingOptionProjectTelemetry());
+
+void Trace::RegisterProvider() noexcept
+{
+    TraceLoggingRegister(g_hProvider);
+}
+
+void Trace::UnregisterProvider() noexcept
+{
+    TraceLoggingUnregister(g_hProvider);
+}
+
+void Trace::Invoked() noexcept
+{
+  TraceLoggingWrite(
+        g_hProvider,
+        "PowerRename::Event::Invoked",
```

Comment:
Hi Chris,
we are in the process of fixing the telemetry event names https://github.com/microsoft/PowerToys/pull/545
As you can see we are currently removing `::Event::` and using the underscore as a separator.